bitcoind-monitor.py

```diff
@@ -228,6 +220,32 @@ def do_smartfee(num_blocks: int) -> None:
         gauge.set(smartfee)
 
 
+def hashps_gauge_suffix(nblocks):
+    if nblocks < 0:
+        return "_NEG%d" % -nblocks
+    if nblocks == 120:
+        return ""
+    return "_%d" % nblocks
```

Comment:
For backwards compatibility reasons with existing metrics, I'd like to avoid the double underscore and use lowercase `neg`.

```suggestion
def hashps_gauge_suffix(nblocks):
    if nblocks < 0:
        return "neg%d" % -nblocks
    if nblocks == 120:
        return ""
    return "%d" % nblocks
```

Reply:
Nice catch, didn't realize this issue. However, this version changes metric `bitcoin_hashps` to `bitcoin_hashps_`. I have performed a slightly different fix here.